README.md

```diff
@@ -25,14 +25,10 @@ application for Google Summer of Code in the current and previous years.
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
 
-- [Students](#students)
-- [Sub Organizations](#sub-organizations)
-- [Organizations Confirmed Under NumFOCUS Umbrella](#organizations-confirmed-under-numfocus-umbrella)
-- [NumFOCUS Projects](#numfocus-projects)
-    - [Fiscally Sponsored Projects GSoC Status](#fiscally-sponsored-projects-gsoc-status)
-    - [Affiliated Projects GSoC Status](#affiliated-projects-gsoc-status)
-    - [Other Projects GSoC Status](#other-projects-gsoc-status)
-- [About GSoC](#about-gsoc)
+- [Google Summer of Code](#google-summer-of-code)
```

Comment:
Please do not change this part in the same PR. I suggest you create another PR for these changes that are not related to submitting your idea list.

This applies to the changes in Zarr line. This will make our work much easier.

Reply:
I didn't actually edit this manually, it was done magically by VSCode. I think the removed links are actually missing in this document?